diff --git a/casatools/src/code/CMakeLists.txt b/casatools/src/code/CMakeLists.txt
index 9f26005..e0cf581 100644
--- a/casatools/src/code/CMakeLists.txt
+++ b/casatools/src/code/CMakeLists.txt
@@ -80,8 +80,9 @@ find_package(FLEX REQUIRED)
 # protobuf/grpc dependency
 find_package(Protobuf REQUIRED)
 find_program(_GRPC_CPP_PLUGIN_EXECUTABLE grpc_cpp_plugin)
-# Try to detect GRPC using native cmake support (for distributions like Fedora or source compilation)
-find_package(gRPC QUIET)
+# conda-forge: As of 2025 April, CMake search errors out incorrectly:
+# https://github.com/protocolbuffers/protobuf/issues/18307
+# find_package(gRPC QUIET)
 # If that fails, try detecting GRPC with pkgconfig (for distributions like Ubuntu, Rocky Linux)
 if(NOT DEFINED gRPC_FOUND OR gRPC_FOUND EQUAL 0)
   pkg_check_modules(GRPC REQUIRED IMPORTED_TARGET grpc++)
