diff --git a/casatools/src/code/synthesis/TransformMachines2/WProjectFT.h b/casatools/src/code/synthesis/TransformMachines2/WProjectFT.h
index ef78135..b6eb821 100644
--- a/casatools/src/code/synthesis/TransformMachines2/WProjectFT.h
+++ b/casatools/src/code/synthesis/TransformMachines2/WProjectFT.h
@@ -51,7 +51,6 @@
 namespace casacore{
 
 template <class K, class V> class SimpleOrderedMap;
-template <class T> class PtrBlock;
 template <class T> class CountedPtr;
 }
 
